saleor/core/models.py

```diff
@@ -143,20 +149,74 @@ class Meta:
         abstract = True
 
 
+class EventPayloadManager(models.Manager["EventPayload"]):
+    @transaction.atomic
+    def create_with_payload_file(
+        self, payload: str, webhook: Optional["Webhook"] = None
+    ) -> "EventPayload":
+        obj = super().create()
+        obj.save_payload_file(payload, webhook)
+        return obj
+
+    @transaction.atomic
+    def bulk_create_with_payload_files(
+        self,
+        objs: Iterable["EventPayload"],
+        payloads: Iterable[str],
+        webhooks: Iterable["Webhook"],
+    ) -> list["EventPayload"]:
+        created_objs = self.bulk_create(objs)
+        for obj, payload_data, webhook in zip(created_objs, payloads, webhooks):
+            obj.save_payload_file(payload_data, webhook)
```

Comment:
In case of fail to save the file, there is a possibility that we will not generate the proper webhook event. In that case, we can lose the data - for example updating fulfillment will happen but we will not trigger a proper event. 
So my question is:  Do we take this into account? How do we want to be sure that we will always send an event?
I see that you didn't delete the `payload` field from the model. In that case, are we still going to use them? Maybe we could treat a `payload` field as a fallback for storing the data?

Reply:
@maarcingebala What do you think. In my option it's good idea to use legacy payload field as fallback for potential storage issues.

Reply:
In this case we can miss potential issues with storage config e.g. missing permissions. Also, legacy `payload` field is going to be deprecated and deleted in next Saleor versions.